api/views/workflow.py

```diff
@@ -36,7 +36,7 @@ class ProviderViewSet(viewsets.ReadOnlyModelViewSet):
     serializer_class = ProviderSerializer
 
     def get_queryset(self):
-        queryset = ShareUser.objects.exclude(robot='').exclude(long_title='')
+        queryset = ShareUser.objects.exclude(long_title='').exclude(favicon='')
```

Comment:
That makes sense! I may have been confused by the title/description of this PR. It's probably close enough to intuitive that in order to be prominently displayed, you need a favicon. I'm good with that.

Is there more than one OSF push user? I thought the other OSF users are just logging into share, and are not trusted.